sormas-api/src/main/resources/captions_de-DE.properties

```diff
@@ -58,6 +58,8 @@ remove=Entfernen
 reportingUser=Meldender Nutzer
 
 # About
+about=About
```

Comment:
Revert changes to this file please; they contain a lot of errors and will be overwritten by Crowdin anyway. Translations aren't taken care of by us. (I know it was necessary to add them for testing purposes)